_slides/02-beardofedu.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+"Making commits without a network connection!" - change this pls
 
 ![jetpacktocat](https://octodex.github.com/images/jetpacktocat.png)
```

Comment:
Maybe you should use this image instead: ![img](https://octodex.github.com/images/filmtocats.png)